JoinColony/colonyDapp
# Implement safe transaction motion page

## Description

This PR wires in the `DefaultMotion` page when creating a motion with one or multiple safe transactions. 

## To test
1. Set up your bridging environment (instructions copied below). 
2. Add a safe
3. Generate reputation before installing the motion extension
4. Create as many different transactions as you like, from 1 to multiple.
5. The detail widget should look like the one we already use on the action page, and the title of the transactions should be displayed on the finalize and vote widgets.
6. On the colony home, the event list should display the motions and their state (Mind you, they should only appear when they are staked up to 10% of the stake).
7. Bonus if you test the other non-safe motions, just to make sure the changes added are not affecting the other motions

## Bridging env instructions

0. have your normal dev env running. 
1. `cd src/lib/colonyNetwork` 
8. `git fetch` 
9. `git checkout maint/cross-chain-safe` 
10. `git submodule update --init` 
11. `yarn` 
12. `yarn provision:safe:contracts` 
13. `cd lib/safe-contracts` then `yarn` (ignore errors)
14. `cd ../..` (back into colonyNetwork) 
15. `bash scripts/start-bridging-environment.sh` 
16. `node scripts/start-bridging-environment.js` 
17. Copy the data in console into your .env and restart webpack (home bridge address, local safe address, local token address etc.)

```
# For testing Safe Control locally. Ensure you have a foreign chain running locally.

LOCAL_HOME_BRIDGE_ADDRESS=

LOCAL_FOREIGN_BRIDGE_ADDRESS=

LOCAL_SAFE_ADDRESS=

ZODIAC_BRIDGE_MODULE_ADDRESS=

LOCAL_ERC721_ADDRESS=

LOCAL_SAFE_TOKEN_ADDRESS=
```


![FireShot Capture 736 - NFT moment - Motion - Colony - wonker - localhost](https://user-images.githubusercontent.com/18473896/199380392-78706f31-624a-415b-a781-b087da4c7be3.png)
![FireShot Capture 737 - Actions - Colony - wonker - localhost](https://user-images.githubusercontent.com/18473896/199380397-7225e384-bec6-4d3d-9d01-3560e316fb43.png)


Resolves #4004
